MDANSE/Src/MDANSE/Framework/Jobs/TrajectoryFilter.py

```diff
@@ -0,0 +1,232 @@
+#    This file is part of MDANSE.
+#
+#    MDANSE is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+import collections
+import copy
+import json
+
+import numpy as np
+
+import h5py
+from MDANSE.Framework.Formats import HDFFormat
+
+from MDANSE.Chemistry.ChemicalEntity import AtomGroup
+from MDANSE.Framework.Jobs.IJob import IJob
+from MDANSE.Mathematics.Signal import filter_map
+from MDANSE.MolecularDynamics.Configuration import RealConfiguration
+from MDANSE.MolecularDynamics.Trajectory import sorted_atoms, TrajectoryWriter
+from MDANSE.MLogging import LOG
+
+
+class TrajectoryFilter(IJob):
+    """
+    Design and apply a filter for the atomic trajectories.
+    """
+
+    label = "Trajectory Filter"
+
+    category = (
+        "Analysis",
+        "Dynamics",
```

Comment:
Maybe the category of this job should be "Trajectory"? @MBartkowiakSTFC do you have any opinions?

Reply:
"Trajectory" makes sense, since it outputs trajectories. I did look there first when I wanted to find this. After that I checked Dynamics next, so it is also not bad. But let's move it to "Trajectory".

We probably need to improve the sorting of the jobs, add some search options, etc. There are already a few jobs with an ambiguous category.